.gitignore

```diff
@@ -9,3 +9,4 @@
 # diameter compiler atrifacts
 /src/diameter_*
 /include/diameter_*
+rebar3
```

Comment:
I thought we talked about this and said that nobody would put rebar3 in the repo?

Reply:
I use `rebar3` binary in repo during development, and for me it is a little inconvenience to care that it is in the root folder or not during commits

Reply:
on Linux, putting in `~/bin` might be more convenient (create the directory if needed)

Reply:
Yes, thanks, I know it, but I use different `rebar` versions and by this reason I not put it into `~/bin`